mne/report.py

```diff
@@ -316,6 +316,49 @@ def _update_html(html, report_fname, report_sectionlabel):
     return htmls, report_fnames, report_sectionlabels
 
 
+def open_report(fname, **params):
+    """Read a saved report or, if it doesn't exist yet, create a new one.
+
+    The returned report can be used as a context manager, in which case any
+    changes to the report are saved when exiting the context block.
+
+    Parameters
+    ----------
+    fname : str
+        The file containing the report, stored in the HDF5 format. If the file
+        does not exist yet, a new report is created that will be saved to the
+        specified file.
+    **params : list of parameters
```

Comment:
Isn't it better to use `copy_function_doc_to_method_doc` to list the parameters and their doc?

Reply:
I suspect it does not work where you have a starting extra `Parameter`